modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -7,7 +7,32 @@ class EvidenceWaiverController < ClaimsApi::V2::ApplicationController
         before_action :verify_access!
 
         def submit
-          render json: { status: 'OK' }
+          bgs_claim = find_bgs_claim!(claim_id: params[:claimId])
+
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found') if bgs_claim.blank?
+
+          render json: { status: 'OK', success: true }
```

Comment:
I feel like I remember seeing a document that showed a response of `success: true`. I was thinking we just go with that to match, but now I can't find said document. I'm not opposed to leaving both in, but seems redundant to have `status: 'OK'` and `success: true` here. I'd vote we just return one.

Reply:
[eb92f10](https://github.com/department-of-veterans-affairs/vets-api/pull/10822/commits/eb92f1054725fd171b879d4554eb6ceba8e2d866)